### DISCUSSION TITLE
Plot Size Issue - No One Pager Plots exporting

### DISCUSSION THREAD

**Discussion by melanie-wander** [facebookexperimental/Robyn#300] [Q&A]
I am getting the below error message for exporting my 1 pager plots, I did the suggested change however  this doesn't seem to be working as I receive the exact same error message. Does anyone know how to get rid of the size limits, per Robyn instruction, I went to the most granular media (platform) but this is causing the issue with size output limit with the 1 page plots.

```
Using robyn object location: model

>>> Running Pareto calculations for 10000 models on 3 fronts...

>>> Collecting 69 pareto-optimum results into: model/2022-02-10 08.00 init/

>> Exporting pareto results as CSVs into directory...

>> Exporting general plots into directory...

Failed exporting results, but returned model results anyways:
 Error in `plot_dim()`:
!   Dimensions exceed 50 inches (height and width are specified in 'in' not pixels).
  If you're sure you want a plot that big, use `limitsize = FALSE`.


Total run time: 26.13 mins
```

```
OutputCollect <- robyn_run(
  InputCollect = InputCollect # feed in all model specification
  , plot_folder = robyn_object # plots will be saved in the same folder as robyn_object
  , pareto_fronts = 3
  , plot_pareto = TRUE
  , limitsize = FALSE  
  )
```

**Comment by gufengzhou** [comment]
Hey, have you updated the package already?  The latest version already sets `limitsize = FALSE`

**Reply by melanie-wander** [reply]
Unfortunately, yes I had forced a latest verison update the day of. According to Facebook MMM documentation more granular media channels are better such as display, uac, and youtube versus just adwords. I would assume that means provisions would have been made to support print of one-page for more media accounts. Do you have any suggestions? I did see there was a release today with more print options which I have not tried out yet.
